contracts/meta-oracles/lib/EMALibrary.sol

```diff
@@ -54,22 +56,13 @@ library EMALibrary{
         uint256 _currentAssetPrice
     )
         internal
-        view
+        pure
         returns (uint256)
     {
-        uint256 weightedMultiplierNumerator = 2;
-        uint256 weightedMultiplierDenominator = _timePeriod.add(1);
-
-        uint256 currentWeightedValue = _currentAssetPrice.getPartialAmount(
-            weightedMultiplierNumerator,
-            weightedMultiplierDenominator
-        );
-
-        uint256 previousWeightedValue = _previousEMAValue.getPartialAmount(
-            weightedMultiplierNumerator,
-            weightedMultiplierDenominator
-        );
+        uint256 a = _currentAssetPrice.mul(2);
```

Comment:
Are we going with this naming? Nothing in our codebase is like this.

Reply:
Mmm, I don't have a better name for it - given its steps in a formula and each step doesn't really mean anything